### PR TITLE
Adding few dev jokes

### PR DESCRIPTION
**Pull request by himanshu1603**
Added new jokes please check them.

### CODE REVIEW

File: src/jokes.json [hunk 1]
```diff
@@ -699,5 +699,15 @@
   },
   "159": "99 little bugs in the code, 99 bugs in the code, 1 bug fixed...compile again, 100 little bugs in the code.",
   "160": "UNIX is very user friendly... It's just very particular about who its friends are.",
-  "161": "Debugging: Removing the needles from the haystack."
+  "161": "Debugging: Removing the needles from the haystack.",
```
Comment (ABSphreak) [inline src/jokes.json:702]: @himanshu1603 resolve conflicts from here, by updating your fork.

**Review by ABSphreak — CHANGES_REQUESTED** [review]

**Comment by ABSphreak** [comment]
@himanshu1603 nice jokes you got there, I will merge them, please resolve the conflicts. 😄

**Comment by himanshu1603** [comment]
Thanks!! resolved the conflicts. Please check the update.

**Comment by ABSphreak** [comment]
Don't forget to 🌟 the repo!
Thanks for your contribution!

**Pull request merged**